index.html

```diff
@@ -0,0 +1,163 @@
+                        <li><i class="fa-brands fa-java"></i> Java</li>
+                        <li> <i class="fa-solid fa-database"></i> Database</li>
+                        <li>   <i class="fas fa-lightbulb"></i> Problem solving</li>
+                        <li><svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Xamarin</title><path d="M6.925 1.323a1.904 1.904 0 00-1.609.93L.241 11.07a1.918 1.918 0 000 1.862l5.075 8.816c.326.56.96.929 1.609.93h10.15a1.904 1.904 0 001.609-.93l5.075-8.816a1.918 1.918 0 000-1.862l-5.075-8.816a1.904 1.904 0 00-1.609-.93zm.092 5.157a.22.22 0 01.043 0h1.75a.23.23 0 01.192.114l2.97 5.292a.228.228 0 01.028.086.228.228 0 01.028-.086l2.963-5.292a.231.231 0 01.198-.114h1.751c.155.002.271.197.199.334L14.239 12l2.9 5.179c.08.138-.04.342-.199.34h-1.75a.232.232 0 01-.2-.12l-2.962-5.292A.228.228 0 0112 12.02a.228.228 0 01-.028.086l-2.97 5.292a.231.231 0 01-.192.12H7.06c-.16.002-.278-.202-.199-.34L9.761 12l-2.9-5.186c-.07-.125.015-.307.156-.334Z"/></svg>  Xamarin Platform</li>
+                    </ul>
+                </div>
+            </section>
+
+            <section id="Projects" class="portfolio__section portfolio__section4">
+                <h2 class="main__section4__h"> My Projects </h2>
+                <div class="portfolio__section main__section4--project">
+                    <div class="main__section4--imagecontainer">
+                        <img class="main__section4--project__img main__section4--project1__img" src="images/project1First.jfif" alt="web project1 image" title="Rock Scissors paper game image" width="350" height="350" loading="lazy">
+                        <img class="main__section4--project__img main__section4--project1__img" src="images/project1Second.jfif" alt="web project1 image" title="Rock Scissors paper game image" width="350" height="350" loading="lazy">
+                        <img class="main__section4--project__img main__section4--project1__img" src="images/project1Third.jfif" alt="web project1 image" title="Rock Scissors paper game image" width="350" height="350" loading="lazy">
```

Comment:
you can change the alt 

```suggestion
                        <img class="main__section4--project__img main__section4--project1__img" src="images/project1Third.jfif" alt="web project1 main image" title="Rock Scissors paper game image" width="350" height="350" loading="lazy">
```